rbs/variables.tf

```diff
@@ -0,0 +1,105 @@
+#
+# General Vars
+#
+variable "dependencies" {
+  # See: https://github.com/hashicorp/terraform/issues/21418#issuecomment-495818852
+  type        = any
+  default     = []
+  description = "Work-around for Terraform 0.12's lack of support for 'depends_on' in custom modules."
+}
+variable "enable" {
+  type        = bool
+  description = "Enable flag for this module. If set to false, no resources will be created."
+  default     = true
+}
+variable "google_project" {
+  type        = string
+  description = "The google project in which to create resources"
+}
+variable "google_folder_id" {
+  type        = string
+  description = "The folder in which RBS has permission."
+  default     = ""
+}
+variable "cluster" {
+  type        = string
+  description = "Terra GKE cluster suffix, whatever is after terra-"
+}
+variable "cluster_short" {
+  type        = string
+  description = "Optional short cluster name"
+  default     = ""
+}
+variable "owner" {
+  type        = string
+  description = "Environment or developer. Defaults to TF workspace name if left blank."
+  default     = ""
+}
+locals {
+  owner   = var.owner == "" ? terraform.workspace : var.owner
+  service = "rbs"
```

Comment:
following the same reasoning, can we entirely remove `service`?

Reply:
Done

Reply:
Actually we need this because other modules are using it, e.g., https://github.com/broadinstitute/terraform-ap-modules/blob/master/crl-janitor/cloudsql.tf#L10

Reply:
We don't need the rbs/variables `service` for something in crl-janitor. I think we should replace the `rbs` usage of `service` in cloudsql with the plain string instead of referenceing `local.service`.

Reply:
Sorry, the crl-janitor link is similar reference RBS will have. 
I feel we should keep the local variable as it a generic template there then we don't need to replace all service name everywhere:
https://github.com/broadinstitute/terraform-ap-modules/blob/yyu-PF-110/rbs/cloudsql.tf#L10
https://github.com/broadinstitute/terraform-ap-modules/blob/yyu-PF-110/rbs/cloudsql.tf#L18
https://github.com/broadinstitute/terraform-ap-modules/blob/yyu-PF-110/rbs/cloudsql.tf#L22
https://github.com/broadinstitute/terraform-ap-modules/blob/yyu-PF-110/rbs/sa.tf#L7
https://github.com/broadinstitute/terraform-ap-modules/blob/yyu-PF-110/rbs/sa.tf#L8
https://github.com/broadinstitute/terraform-ap-modules/blob/yyu-PF-110/rbs/sa.tf#L7

Reply:
I don't think it's hard to find/replace service names, and I don't know how much copy/pasting we will be doing from/to this module after this initial commit. I think there's some value in being explicit in the names, but not huge deal.

Reply:
Yeah, agree. But, there is also value of having common variables, just like our regular code:
```String SERVICE_NAME = "rbs"```
And other place can refer to this something like 
```SERVICE_NAME + "db",  `SERVICE_NAME + "db_name",``` because SERVICE_NAME is used everywhere across the module?

Reply:
Common variables are useful for
a) documentation (but here the module name tells us this is all about `rbs`)
b) being able to change something in one place and affect many.  (But we're not likely to rename our service, but it also wouldn't be hard to do later).
c) Making sure different places always use the same value when they need to. (This is not the case for `service_name + "db"` and anything else using the service name. If those values changed independently that wouldn't break anything).
So meh.

Reply:
Fine with ethier way. But somehow I was implied that McTerra follows similar naming modules, service name as main db, service-stairway as stairway db, hostname is service name, service-{SA role} as SA account naming. And this is the default conversion and falls into c) ?

Reply:
let's just leave it.

Reply:
I don't have a super strong opinion on this, slightly leaning towards leaving it as is, but ultimately it's up to y'all. AFAIK changing it won't break anything.